<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Homepage</class>
 <widget class="QMainWindow" name="Homepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HOME</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image : url(image.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="upload_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(146, 208, 80)</string>
    </property>
    <property name="text">
     <string>Upload File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="patientData_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(146, 208, 80)</string>
    </property>
    <property name="text">
     <string>View Patient Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editInfo_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(146, 208, 80)</string>
    </property>
    <property name="text">
     <string>EDIT PATIENT INFO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetDatabase_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(154, 0, 0);
color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>RESET DATABASE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editValues_btn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>450</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(84, 84, 84);
color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Edit Values</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
